packages/shader-lab/src/preprocessor/PpParser.ts

```
@@ -105,7 +105,9 @@
 
   private static reportError(loc: ShaderRange | ShaderPosition, message: string, source: string, file: string) {
     const error = ShaderLabUtils.createGSError(message, GSErrorName.PreprocessorError, source, loc, file);
+    // #if _VERBOSE
     this._errors.push(error);
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)